jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -166,9 +262,9 @@ YUI.add('user-profile', function() {
             <div className="inner-wrapper">
               <juju.components.UserProfileHeader
                 avatar=""
-                bundleCount={0}
-                charmCount={0}
-                environmentCount={this.state.envList.length}
+                bundleCount={bundleCount || 0}
```

Comment:
Yeah, I agree. Originally the definition got pretty hairy on me so I moved the "or 0" bit down here, but if I make the "helps minification" change then I think the "or 0" can be moved back up.